README.rst

```diff
@@ -825,10 +825,9 @@ The source distribution contains:
    somewhere into your PYTHONPATH. Alternatively, you can also run
    the distutils scripts: ``python setup.py install``
 
--  ``test``: some unit and integrations tests. In order to run the tests 
-   some packages have to be installed before. So please install the packages 
-   listed in requirements.development.txt:
-   ``pip install -r requirements.development.txt``
+-  ``test``: some unit and integrations tests. In order to run the tests
```

Comment:
I'm fairly sure this is deprecated:

- https://github.com/pytest-dev/pytest/issues/5534
- https://docs.pytest.org/en/latest/explanation/goodpractices.html#integrating-with-setuptools-python-setup-py-test-pytest-runner
  > Integration with setuptools is not recommended, i.e. you should not be using python setup.py test or pytest-runner, and may stop working in the future.

Maybe it is okay for now, but better to point people to tox if they want something easy to manage.

I'm somewhat okay with the change you made as is, because it makes it no worse, but this is something we have to fix sometime.